metals/src/main/scala/scala/meta/internal/builds/SbtBuildTool.scala

```diff
@@ -25,7 +25,7 @@ case class SbtBuildTool(
 
   override def args(workspace: AbsolutePath): List[String] = {
     val sbtArgs = List[String](
-      "set bloopExportJarClassifiers in Global := Some(Set(\"sources\"))",
+      "-Dbloop.export-jar-classifiers=\"sources\"",
```

Comment:
Do we want the extra quote signs `\"`?

Reply:
It seems to not work in the Sbt integration tests currently. Might be because of of the additional quotes?

Reply:
Happy new year!
I'm back from holidays and going to work on this as soon as possible!